server/routes/replayEventRouter.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const replayEventRouter = express.Router();
+
+const { handleEvent, getDomainEvents, dispatch } = require('../models/DomainEvent');
+
+const Session = require('../infra/database/Session');
+const { publishMessage } = require('../infra/messaging/RabbitMQMessaging');
+
+const { EventRepository } = require('../infra/database/PgRepositories');
+const { handleVerifyCaptureProcessed } = require('../services/EventHandlers');
+
+replayEventRouter.post("/", async function(req, res) {
+    const session = new Session(false);
+    const eventRepository = new EventRepository(session);
+    const executeGetDomainEvents = getDomainEvents(eventRepository);
+    const domainEvents = await executeGetDomainEvents({"status":req.body.status});
+    if(req.body.status == "raised"){
+        const eventDispatch = dispatch(eventRepository, publishMessage);
+        try {
+            domainEvents.forEach(async (domainEvent) => {
+                const result = await executeGetDomainEvents({"id":domainEvent.id});
+                eventDispatch(domainEvent);
+            });
+        } catch(e) {
+            console.log(e);
+            if (session.isTransactionInProgress()){
+                await session.rollbackTransaction();
+            }
+            if (migrationSession.isTransactionInProgress()) {

```

Comment:
The migrationSession is not needed for the replay events logic. This is because events are new for the services and are not present in the legacy database. So we could omit this section.